ParckingApplication.Tests/UnitTest1.cs

```diff
@@ -8,25 +16,66 @@ public class UnitTest1
     public void Test1()

```

Comment:
Redenumeste testele in formatul cerut si de Ovi initial ( *functia testata*_*rezultatul asteptat*)